packages/react-core/src/components/Alert/AlertActionCloseButton.tsx

```diff
@@ -35,3 +35,4 @@ export const AlertActionCloseButton = ({
     )}
   </AlertContext.Consumer>
 );
+AlertActionCloseButton.displayName = 'AlertActionCloseButton';
```

Comment:
On line 17 can we type `AlertActionCloseButton` as a `React.FunctionComponent<AlertActionCloseButtonProps>` like all our other components as well? That's why the linter didn't pick it up...

Reply:
Done :+1: